# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/RemoteContentController.h
# Commit: 4230bdcf07ae
# Full Hash: 4230bdcf07ae69e03a9e241f10c201c38e6ca995
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r d1ddb1ce916b -r 4230bdcf07ae gfx/layers/ipc/RemoteContentController.h
--- a/gfx/layers/ipc/RemoteContentController.h	Wed Jun 22 16:08:22 2022 +0000
+++ b/gfx/layers/ipc/RemoteContentController.h	Wed Jun 22 16:15:57 2022 +0000
@@ -81,6 +81,9 @@
 
   void CancelAutoscroll(const ScrollableLayerGuid& aScrollId) override;
 
+  void NotifyScaleGestureComplete(const ScrollableLayerGuid& aGuid,
+                                  float aScale) override;
+
   void ActorDestroy(ActorDestroyReason aWhy) override;
 
   void Destroy() override;
@@ -106,6 +109,10 @@
 
   void CancelAutoscrollInProcess(const ScrollableLayerGuid& aScrollId);
   void CancelAutoscrollCrossProcess(const ScrollableLayerGuid& aScrollId);
+  void NotifyScaleGestureCompleteInProcess(const ScrollableLayerGuid& aGuid,
+                                           float aScale);
+  void NotifyScaleGestureCompleteCrossProcess(const ScrollableLayerGuid& aGuid,
+                                              float aScale);
 };
 
 }  // namespace layers
